src/Spectre.libGaussianMixtureModelling/RandomInitializationRef.h

```diff
@@ -137,8 +144,8 @@ namespace Spectre::libGaussianMixtureModelling
         }
 
         DataType* m_pMzArray;
-        unsigned int m_DataSize;
+        unsigned m_DataSize;
```

Comment:
Since we stick to `unsigned`, it would be good to introduce this to coding standard.